pkg/resources/secrets/etcd.go

```diff
@@ -26,7 +26,9 @@ type Etcd struct {
 
 // EtcdCertsSpec describes etcd certs secrets.
 type EtcdCertsSpec struct {
-	EtcdPeer *x509.PEMEncodedCertificateAndKey `yaml:"etcdPeer"`
+	EtcdPeer      *x509.PEMEncodedCertificateAndKey `yaml:"etcdPeer"`
```

Comment:
do we need `EtcdPeer` and `EtcdClient` here?

Reply:
get etcdsecrets.secrets.talos.dev -oyaml -- looks nice
And you can check all certs. 

https://github.com/talos-systems/talos/pull/3777/files#diff-f028d821b1f248fa75f380f9a91477df94583726a996ead5f16b089ec24baf04R130 here updateSecrets func recreate the certs